hubspot-style/src/main/java/com/hubspot/immutables/style/HubSpotStyle.java

```diff
@@ -9,11 +9,15 @@
 import org.immutables.value.Value.Style.ImplementationVisibility;
 
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.hubspot.immutable.collection.encoding.ImmutableListEncodingEnabled;
+import com.hubspot.immutable.collection.encoding.ImmutableSetEncodingEnabled;
 import com.hubspot.immutables.validation.InvalidImmutableStateException;
 
 @Target({ ElementType.PACKAGE, ElementType.TYPE})
 @Retention(RetentionPolicy.CLASS) // Make it class retention for incremental compilation
 @JsonSerialize
+@ImmutableSetEncodingEnabled
+@ImmutableListEncodingEnabled
```

Comment:
Missing the map encoding enabler here?

Reply:
that's intentional, the map encoding is more risky b/c of how its behavior differs from a standard map (`put` with a dupe key throws in `ImmutableMap`), this is why we set `jdkOnly = true`, so I will have to figure out a safe rollout plan for that.